src/pages/Introduce/Introduce.scss

```
@@ -0,0 +1,34 @@
+@import "/src/style/variables.scss";
+@import "/src/style/mixin.scss";
+
+.Introduce {
+  @include flexcenter;
+  margin-top: 300px;
+
+  .Introduceform {
+    border-style: solid;
+    border-width: 3px;
+    border-radius: 10px;
+    height: 300px;
+    width: 250px;
+    @include textStyle(13, 13px);
```

```suggestion
    @include textStyle(13, 13px);
    border-style: solid;
    border-width: 3px;
    border-radius: 10px;
    height: 300px;
    width: 250px;
```

mixin 으로 만든 스타일묶음을 include로 가져올 때
스타일에 가장 상위에 위치시키는 것이 규칙입니다~